recyclerview/recyclerview/src/main/java/com/microsoft/device/dualscreen/recyclerview/utils/DeltaCalculator.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.dualscreen.recyclerview.utils
+
+import android.view.View
+import com.microsoft.device.dualscreen.utils.wm.getWindowRect
+
+internal class DeltaCalculator {
```

Comment:
Maybe it would be useful to add a comment describing what this delta calculates?

Reply:
Done